### PR TITLE
Add hook pyshark

### PR DESCRIPTION
**Pull request by LpCodes**
Have made changes as suggest kindly review

### CODE REVIEW

**Review by rokm — CHANGES_REQUESTED** [review]
See the inline comments. Also, make your changes to this PR instead of opening a new one. Thanks!

File: src/_pyinstaller_hooks_contrib/tests/test_libraries.py [hunk 1]
```diff
@@ -1196,3 +1196,14 @@ def test_great_expectations(pyi_builder):
         from great_expectations.render.view import view
         v = view.DefaultJinjaView()
         """)
+
```
Comment (rokm) [inline src/_pyinstaller_hooks_contrib/tests/test_libraries.py:1199]: There must be two empty lines between individual tests (this is part of why linter check is failing).

**Review by rokm — CHANGES_REQUESTED** [review]

File: src/_pyinstaller_hooks_contrib/tests/test_libraries.py (hunk 1 above)
Comment (LpCodes) [reply src/_pyinstaller_hooks_contrib/tests/test_libraries.py:1213]: Added 2 empty lines

**Comment by bwoodsend** [comment]
Thanks @LpCodes!

**Pull request merged**